api/schema.yml

```diff
@@ -298,6 +298,24 @@ paths:
       responses:
         "204":
           description: No response body
+  /api/v2/files/{id}/delete_file:
```

Comment:
I'd prefer to have this at `DELETE /api/v2/files/{id}`

Reply:
That's what i had in my first attempt. I think this is the result of a brainfart on my part. lemme recheck the code.

Reply:
If you look at the view class we inherited from, you have the `DestroyModelMixin`. You can override the `perform_destroy` function:
```
class DestroyModelMixin:
    """
    Destroy a model instance.
    """
    def destroy(self, request, *args, **kwargs):
        instance = self.get_object()
        self.perform_destroy(instance)
        return Response(status=status.HTTP_204_NO_CONTENT)

    def perform_destroy(self, instance):
        instance.delete()
```

Reply:
Had to override destroy itself because the perform_destroy does not return results but its working now. Need to run more manual tests today.